lib/ai_bot/tools/random_picker.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module AiBot
+    module Tools
+      class RandomPicker < Tool
+        def self.signature
+          {
+            name: name,
+            description:
+              "Handles a variety of random decisions based on the format of each input element",
+            parameters: [
+              {
+                name: "options",
+                description:
+                  "An array where each element is either a range (e.g., '1-6') or a comma-separated list of options (e.g., 'sam,jane,joe')",
+                type: "array",
+                item_type: "string",
+                required: true,
+              },
+            ],
+          }
+        end
+
+        def self.name
+          "random_picker"
+        end
+
+        def options
+          parameters[:options]
+        end
+
+        def invoke(_bot_user, _llm)
+          result = nil
+          # can be a naive list of strings
+          if options.all? { |option| !option.match?(/\A\d+-\d+\z/) && !option.include?(",") }
```

Comment:
```suggestion
          if options.none? { |option| option.match?(/\A\d+-\d+\z/) || option.include?(",") }
```

Nitpicky, but I think it reads better if we do the opposite. Goes from "any none of these" to "none of either of these"

Reply:
agree, reads better